dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -221,7 +221,7 @@ public void SpecifiedPlacement()
                     SetHeroContent(HeroContentOptions.NoContent);
 
                     var targetRect = GetTargetBounds();
-
+                    var isRTL = location == TipLocationOptions.ResourceDictionary;
                     // All positions are valid
```

Comment:
there also needs to be a test that sets the FlowDirection on a common ancestor between the TeachingTip and the target. I'm worried that the forcing of the Popup child to be LeftToRight breaks the case where the window root is RTL.

Reply:
I did all of my development with RTL set on the page, but i agree having a button to switch flow direction would be useful for testing.